pkg/apis/events/v1beta1/cloudauditlogssource_types.go

```diff
@@ -68,12 +68,20 @@ var auditLogsSourceCondSet = apis.NewLivingConditionSet(
 )
 
 const (
-	CloudAuditLogsSourceEvent = "com.google.cloud.auditlog.event"
+	// Ref: https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/cloud/audit/v1/events.proto#L27
+	// TODO: somehow reference the proto value directly.
+	CloudAuditLogsSourceLogWrittenEventType = "google.cloud.audit.log.v1.written"
+	CloudAuditLogsSourceEventDataSchema     = "https://github.com/googleapis/google-cloudevents/blob/master/proto/google/events/cloud/audit/v1/events.proto"
```

Comment:
can we put all of these constants in `register.go` maybe? outside v1beta1... When we do v1, we will be copy/pasting them otherwise. Similar thing happened with the current values... They are both in v1alpha1 and v1beta1

Reply:
That's a good point!

Reply:
I actually moved this out of the `apis` folder. I feel it makes more sense as they eventually should be replaced by the code generated from the protos.